test/blackbox-tests/test-cases/no_dynlink.t

```diff
@@ -0,0 +1,37 @@
+  $ cat >dune-project <<EOF
+  > (lang dune 3.17)
+  > EOF
+
+  $ cat >dune <<EOF
+  > (library
+  >  (name mylib))
+  > EOF
+
+  $ touch a.ml
+
+  $ dune build --display short
```

Comment:
Could you replace the --display short with just an explicit target such as
mylib.cmxs?

This extra output makes the tests fragile and hard to understand

Reply:
Fixed, thanks.